composer.json

```diff
@@ -19,8 +19,8 @@
         "ext-json": "*",
         "codeception/codeception": "^5.0.8",
         "codeception/lib-xml": "^1.0",
-        "justinrainbow/json-schema": "~5.2.9",
-        "softcreatr/jsonpath": "^0.8"
+        "justinrainbow/json-schema": "^5.2.9",
+        "softcreatr/jsonpath": "^0.8 || ^0.9"
```

Comment:
doesn't `"^0.8"` already include `"^0.9"` ?

Reply:
No, not for pre-1.0 versions (see https://getcomposer.org/doc/articles/versions.md#caret-version-range-):

> For pre-1.0 versions it also acts with safety in mind and treats ^0.3 as >=0.3.0 <0.4.0 and ^0.0.3 as >=0.0.3 <0.0.4.